inventory/sample/group_vars/k8s-cluster/addons.yml

```diff
@@ -17,6 +17,15 @@ metrics_server_enabled: false
 # metrics_server_metric_resolution: 60s
 # metrics_server_kubelet_preferred_address_types: "InternalIP"
 
+# Rancher Local Path Provisioner
+local_path_provisioner_enabled: false
+# local_path_provisioner_namespace: "local-path-storage"
+# local_path_provisioner_storage_class: "local-path"
+# local_path_provisioner_reclaim_policy: Delete
+# local_path_provisioner_claim_root: /opt/local-path-provisioner/
+# local_path_provisioner_debug: false
+# local_path_provisioner_image_tag: v0.0.2
```

Comment:
`# local_path_provisioner_image_repo: "rancher/local-path-provisioner"` should be added here too

Reply:
Done: https://github.com/kubernetes-sigs/kubespray/pull/4232/commits/a36faac33b2f9967ac0816f9e9154668018828a9

Reply:
And done: 
https://github.com/kubernetes-sigs/kubespray/pull/4232/commits/536d29dfb4a47a20fd24dde4a6c6e5161ae2a133